README.md

```diff
@@ -5,6 +5,8 @@
 Allows you to send logs to the [Insight Platform](https://www.rapid7.com/products/)
 (or Logentries) account from Node.js.
 
+This client is not backwards-compatible with [Le_Node](https://github.com/rapid7/le_node).
```

Comment:
```suggestion
This client is not backwards-compatible with [Le_Node](https://github.com/rapid7/le_node) and will require you to make some modifications to your implementation.
```

Reply:
Would be ideal, but there is so much that's changed.
Every package updated, how we generate Transports etc.
Don't think we'd ever hit everything and I can't imagine an easy switchover.